tests/data/constAngularVelocityLineScan.json

```diff
@@ -54,15 +54,15 @@
       [1000.0, 0.0, 0.0]
     ],
     "velocities": [
-      [0.0, 0.0, 0.0],
-      [0.0, 0.0, 0.0],
-      [0.0, 0.0, 0.0],
-      [0.0, 0.0, 0.0],
```

Comment:
Any particular reason these got changed to 10s?

Reply:
See #173 
The velocity isn't actually used besides defining cross-track and down-track. So, the velocity doesn't change the intersections or anything else.